test/merchant_test.rb

```diff
@@ -1,34 +1,27 @@
 require 'simplecov'
 SimpleCov.start
-require './lib/merchant'
-require 'rspec'
-
-RSpec.describe Merchant do
+require './lib/helper'
+
+RSpec.describe InvoiceRepository do
+  let!(:sales_engine) {SalesEngine.from_csv({
+    :items     => "./data/items.csv",
+    :merchants => "./data/merchants.csv"
+  })}
+  let!(:merchant_repository) {sales_engine.merchants}
+  let!(:merchant) {merchant_repository.create(Merchant.new({
+    :id => 5,
+    :name => "Turing School"
+  }))}
 
   it 'exists' do
-    merchant = Merchant.new({
-      :id => 5,
-      :name => "Turing School"
-      })
-
-    expect(merchant).to be_instance_of(Merchant)
+    expect(merchant_repository).to be_instance_of(Merchant)
   end
 
   it 'returns id' do
-    merchant = Merchant.new({
-      :id => 5,
-      :name => "Turing School"
-      })
-
     expect(merchant.id).to eq(5)
   end
 
   it 'returns name' do
```

Comment:
Same as above! 'merchant' seems to be an array, so it's not letting us call .name on it. 

![image](https://user-images.githubusercontent.com/17027357/171972035-39ebbeaa-a28c-4f08-8826-fe32a0b59189.png)

Reply:
fixed